doc/digital-receipt/implementation/digital-receipt-implementation.md

```diff
@@ -0,0 +1,667 @@
+
+![sign+pos-api-helper](https://github.com/fiskaltrust/interface-doc/assets/124153755/b4496ff6-5040-4d5f-90ae-45d03abab862)
+
+The Point of Sale software calls the Middleware's sign endpoint with a regular receipt request - the request will be processed by the fiskaltrust.Middleware. After this step, the POS software receives the receipt response from the fiskaltrust.Middleware (which also contains the data for creating a printed receipt). The Point of Sale software extracts the ftQueueId and ftQueueItemId properties from the receipt response and generates the link for the QR-Code out of this dataset. Final step is the visualization of the QR-Code containing the URL to the digital receipt on the customer display, handheld, self-checkout or any other suitable devices.
+
+The consumer accesses the receipt by scanning the QR-Code displayed on the customer-facing display/device with their mobile phone. The consumer requests the receipt from fiskaltrust and receives an HTML document as the receipt. 
+
+<details>
+<summary>Response details (shortened):</summary> 
+  
+```
+"{
+"ftCashBoxID":"124358e8-9cbd-4332-9076-7ed8b72306ac"
+"ftQueueID":"c719b59a-9ff0-891a-98a1-030776d8c46f"
+"ftQueueItemID":"f838d7c8-3113-4c3a-914a-46d7de555d11"
+"ftQueueRow":"41"
+"cbTerminalID":"T2"
+"cbReceiptReference":"pos-action-identification-02"
+"ftCashBoxIdentification":"bonstart"
+"ftReceiptIdentification":"ft27#IT11"
+"ftReceiptMoment":"2023-08-03T13:12:56.3989064Z"
+}"
+```
+</details>
+
+## Configure POS API Helper 
+
+The POS API Helper is available in all countries. This Helper is responsible for uploading data from the local Queue to the digital receipt endpoint. This Helper is configured in the fiskaltrust.Portal and assigned to each CashBox that uses digital receipts. The POS API Helper changes to an direct upload behavior of the digital receipts.
+
+To proceed with the configuration, login to your fiskaltrust.Portal account first. 
+
+### Queue 
+
+| Step  | Description |
+| ------------- | ------------- |
+| 1  | Navigate to the configuration section and go to Queue  |
+| 2  | Configure Queue  |
+| 3  | Copy the URLs to your local machine (Required for CashBox configuration later)   |
+| 4  | For all countries: Change port to the next free port (+1) and <br/> a.	if no suffix exists after the port: add the suffix "/name_queue" to the URL ("name" can be freely chosen) <br/> b.	if suffix already exists: add the suffix "_queue" to the URL  |
+| 5  | Germany & France only: Change grpc port to the next free port (if port is free no need to go up to the next free port) and add the suffix "/name_queue" to the URL ("name" can be freely chosen)  |
+| 6  | Save changes  |
+
+### Helper 
+
+| Step  | Description |
+| ------------- | ------------- |
+| 1  | Navigate to Helper  |
+| 2  | Create new helper  |
+| 3  | Add description  |
+| 4  | Select  package name "fiskaltrust.service.helper.posapi"   |
```

Comment:
The package is called differently in v1.3 countries (DE and IT): fiskaltrust.Middleware.Helper.PosAPI